starknet_devnet/blueprints/rpc/structures/responses.py

```diff
@@ -200,8 +195,10 @@ async def txn_type() -> RpcTxnType:
         "transaction_hash": rpc_felt(txr.transaction_hash),
         "actual_fee": rpc_felt(txr.actual_fee or 0),
         "status": status(),
-        "block_hash": rpc_felt(txr.block_hash) if txr.block_hash is not None else None,
-        "block_number": txr.block_number or None,
+        "block_hash": rpc_felt(txr.block_hash)
+        if txr.block_hash is not None
+        else rpc_felt(0),
+        "block_number": txr.block_number if txr.block_number is not None else 0,
```

Comment:
Should it be set to `0` as a default?

Reply:
It should have some value (rpc spec from main branch even defines block_hash and block_number as required parameters) and I though 0 is a reasonable default since it has to be a non negative value.

Reply:
Don't you think it should be set to "latest"?